firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/AppDistroTestApplication.kt

```diff
@@ -5,6 +5,8 @@ import android.app.Application
 class AppDistroTestApplication : Application() {
     override fun onCreate() {
         super.onCreate()
-        ShakeForFeedback.enable(this)
+
+        // Default feedback triggers can also be initialized here
+//        ShakeForFeedback.enable(this)
```

Comment:
Remove this line? Or add commented-out code for the `ScreenshotDetectionFeedbackTrigger` as well?

Reply:
Stay tuned for the next PR :)

Right now the ScreenshotDetectionFeedbackTrigger cannot be enabled at the application level, but I will make it so in the next PR.